src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/TestData/GraphWebServiceDefinition_Dogfood.json

```diff
@@ -0,0 +1,190 @@
+﻿{
+  "properties": {
+    "title": "ServiceTitle",
+    "description": "This is some service description",
+    "readOnly": "false",
+    "keys": {
+      "primary": "S8ayf24PAsbQqy8Nm67uGf3MbOeorYKPY8QfXen2+JsBZV5Q0NFfk9tgC7mjJ3aCsUPY8yKCFVcI/sCcOlmkKw==",
```

Comment:
Are these real credentials?  Do we need to define these explicitly in this fashion?